src/app/charts/chart.component.ts

```diff
@@ -69,6 +89,7 @@ export class ChartComponent implements OnChanges {
             // Really, this should a user selectable option
             time_aggregation: this.chart.indicator.valid_aggregations[0]
         };
+        this.dateRange = [1950, 2100]; // reset time slider range
```

Comment:
Might be good to move these years to named constants. Do these relate to specific dates in the API project? If so, might want to note here and over there that if one set of dates gets changed, the other should be, too.